src/components/Molecules/Membership/Membership.style.js

```diff
@@ -0,0 +1,152 @@
+import styled from 'styled-components';
+
+import Input from '../../Atoms/Input/Input';
+import Text from '../../Atoms/Text/Text';
+// import Link from '../../Atoms/Link/Link';
+import spacing from '../../../theme/shared/spacing';
+import { media } from '../../../theme/shared/size';
+
+const Wrapper = styled.div`
+  max-width: 320px;
+  text-align: center;
+
+  ${media('small')} {
+    max-width: 420px;
+    padding: 0 ${spacing('l')};
+  }
+`;
+
+const Header = styled.div`
+  padding: 0 ${spacing('md')};
+  margin-bottom: ${spacing('l')};
+`;
+
+const FormWrapper = styled.div`
+  box-shadow: 0px ${spacing('md')} ${spacing('xl')} rgba(0, 0, 0, 0.3);
+  height: 450px;
+
+  ${media('small')} {
+    height: 430px;
+  }
+`;
+
+const Error = styled(Text)`
```

Comment:
Can we use Text component directly?

Reply:
I may add this icon later, or is it ok to display error message without it
![Screen Shot 2019-10-09 at 11 39 34](https://user-images.githubusercontent.com/30389858/66474605-9273ea00-ea89-11e9-8921-2937f196bebc.png)

Reply:
later we can add the Icon